Cabal/Distribution/Simple/Configure.hs

```diff
@@ -42,6 +42,7 @@ module Distribution.Simple.Configure (configure,
                                       getInstalledPackages,
                                       getInstalledPackagesMonitorFiles,
                                       getPackageDBContents,
+                                      checkPackageDBs,
```

Comment:
Well, a safe place to put it is cabal-install; this way it is not publically exported and can be refactored at will. If it's discovered that it's needed elsewhere we can subsequently move it.